core/sql/exp/ExpHbaseInterface.cpp

```diff
@@ -271,7 +271,12 @@ Lng32 ExpHbaseInterface::coProcAggr(
 
 char * getHbaseErrStr(Lng32 errEnum)
 {
-  return (char*)hbaseErrorEnumStr[errEnum - (Lng32)HBASE_MIN_ERROR_NUM];
+  Lng32 lv_errEnum;
+  if (errEnum < HBASE_MIN_ERROR_NUM || errEnum >= HBASE_MAX_ERROR_NUM)
+     lv_errEnum = HBASE_GENERIC_ERROR; 
```

Comment:
Yes. ERRORCOUNTER hbase table was getting created and this resulted in core dump during my unit testing. Hence the above change. There has been violation of the return codes spread all through the JNI layer. To get around this violation, I have mapped any return codes that didn't fall into known error code range into a generic error code for time being. This needs to be cleaned up later.